CMakeLists.txt

```diff
@@ -290,9 +282,8 @@ if(ENABLE_GCOV)
   message(STATUS "GCOV enabled")
 endif()
 
-set(CMAKE_CXX_FLAGS_RELEASE " -DNDEBUG -s -O2")
-set(CMAKE_CXX_FLAGS_DEBUG   " -DDEBUG -g3 -ggdb3")
-
+# TODO(AK): check current OS here
+add_definitions(-DOS_POSIX) # see line :360
```

Comment:
@mkorniichuk line 360 : 
```
if (HMI STREQUAL "web" AND NOT DEFINED HMIADAPTER)
```
Please avoid line numbers in comments

Reply:
@LuxoftAKutsan fixed within a89e4042f9a29e6b3ec306046c57bd2bd2e2c4f7